frontend/src/pages/profile/libs/enums/activity-breakpoints.enum.ts

```diff
@@ -0,0 +1,8 @@
+const ActivityBreakpoints = {
+  breakpoint1: 'breakpoint1',
+  breakpoint2: 'breakpoint2',
+  breakpoint3: 'breakpoint3',
+  breakpoint4: 'breakpoint4',
+} as const;
```

Comment:
Also please rename filename (remove s from breakpoint**s**
```suggestion
const ActivityBreakpoint = {
  FIRST: 'first_breakpoint',
  SECOND: 'second_breakpoint',
  THIRD: 'third_breakpoint',
  FOURTH: 'fourth_breakpoint',
} as const;
```

Reply:
Maybe it'll be better to use camel case? Because I use this naming in css

Reply:
You are targeting css classes through enum values - so I guess it's the fact that it shouldn't be used directly